Tests/UT.Remove-AzSqlIpException.Tests.ps1

```diff
@@ -0,0 +1,44 @@
+Set-Location $PSScriptRoot\..\Infrastructure-Scripts\
+
+Describe "Remove-AzSqlIpException Unit Tests" -Tags @("Unit") {
+    $Params = @{
+        WhatsMyIpUrl      = "https://not-a-real-web.com"
+        ServerName        = "das-myserver"
+        ResourceGroupName = "das-foo-rg"
+        Name              = "rule1"
+    }
+
+    Mock Get-AzSqlServerFirewallRule -MockWith {
+        return @{
+            "ResourceGroupName" = "das-foo-rg"
+            "Servername"        = "das-myserver"
+            "FirewallRuleName"  = "rule1"
+        }
+    }
+
+    Mock Remove-AzSqlServerFirewallRule -MockWith {
+        return @{
+            "ResourceGroupName" = "das-foo-rg"
+            "Servername"        = "das-myserver"
+            "FirewallRuleName"  = "rule1"
+        }
+    }
+
+    Context "Whats My Ip Service doesn't return a response" {
```

Comment:
This Context is mocking Invoke-RestMethod giving a response of a valid IP address